TextFormat.md

```diff
@@ -1,64 +1,76 @@
 # Text Format
 
-The purpose of this text format is to support:
-
-* View Source on a WebAssembly module, thus fitting into the Web (where every
-  source can be viewed) in a natural way.
-* Presentation in browser development tools when source maps aren't present
-  (which is necessarily the case with [the Minimum Viable Product (MVP)](MVP.md)).
-* Writing WebAssembly code directly for reasons including pedagogical,
-  experimental, debugging, optimization, and testing of the spec itself.
-
-The text format is equivalent and isomorphic to the [binary format](BinaryEncoding.md).
```

Comment:
Although this was inaccurate, we need to keep a statement like it in some variation. In particular, it is no longer clear from the new text that the text format is supposed to map _unambiguously_ to a _unique_ binary (modulo LEB encoding liberties), which is the most important design consideration of all.